draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -120,11 +129,15 @@ class GuidanceBlock extends React.Component<
     )
   }
 
-  bannerClassName(): string {
-    return classnames(styles.banner, {
-      [styles.hidden]: this.state.hidden,
-      [styles.prominent]: this.props.variant === "prominent",
-    })
+  bannerClassName(noMaxWidth): string {
+    return classnames(
+      styles.banner,
+      {
+        [styles.hidden]: this.state.hidden,
+        [styles.prominent]: this.props.variant === "prominent",
+      },
+      noMaxWidth && styles.noMaxWidth
```

Comment:
```suggestion
        [styles.prominent]: this.props.variant === "prominent",
        [styles.noMaxWidth]: noMaxWidth
      },
```
nit/ although this works as intended, this is the pattern that we usually use alongside classnames when a style is conditional.